examples/erc1155/tests/erc1155.rs

```diff
@@ -16,14 +23,28 @@ fn random_values(size: usize) -> Vec<U256> {
     (0..size).map(|_| U256::from(rand::random::<u128>())).collect()
 }
 
+fn ctr(uri: &str) -> constructorCall {
+    constructorCall { uri_: uri.to_owned() }
+}
+
+const URI: &str = "https://github.com/OpenZeppelin/rust-contracts-stylus";
+
 // ============================================================================
 // Integration Tests: ERC-1155 Token
 // ============================================================================
 
 #[e2e::test]
 async fn constructs(alice: Account) -> eyre::Result<()> {
-    let contract_addr = alice.as_deployer().deploy().await?.address()?;
-    let _contract = Erc1155::new(contract_addr, &alice.wallet);
+    let contract_addr = alice
+        .as_deployer()
+        .with_constructor(ctr(URI))
```

Comment:
I would suggest adding `with_default_constructor()` as we have e.g. in [Erc20](https://github.com/OpenZeppelin/rust-contracts-stylus/blob/main/examples/erc20/tests/erc20.rs#L46); Applies to all E2E tests here.